### DISCUSSION TITLE
How to dump SQL Server Databse in DBeaver

### DISCUSSION THREAD

**Discussion by Brian-wlj** [dbeaver/dbeaver#11175] [Q&A] [answered]
I am using the latest version of DBeaver 7.3.3 and connect my Sql server DB, then I want to dump the DB.
So I right click this DB -> tool , but there is no "Dump" or "Restore" in the menu.

Appreciate your comment on this?

**Comment by Brian-wlj** [comment]
![DBeaver](https://user-images.githubusercontent.com/78243232/106354326-034cae80-632c-11eb-8123-4ae0fd379477.PNG)
see the attached screenshot

**Comment by tati-kru** [comment, marked as answer]
Unfortunately, DBeaver doesn't support backup-restore for SQL Server. You can create a feature request for this.

**Reply by Brian-wlj** [reply]
@tati-kru , thank you very much for reply.  Understood. I thought probably it not suppot SQL dump. just confirm it.
